apps.json

```diff
@@ -2127,7 +2129,7 @@
     "mobilizon": {
         "branch": "master",
         "category": "social_media",
-        "level": 8,
+        "level": 1,
```

Comment:
Seems to be a weird network fluke during the tests, I restarted it, we'll see:

https://ci-apps.yunohost.org/ci/job/1435